.github/workflows/go.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16
```

Comment:
super nit (feel free to ignore): It would be nice to separate these types of changes into their own commit so viewing the overall diff outside of vendor changes doesn't result in a decent lag in the github UI.